app/javascript/packs/order_screen.js

```diff
@@ -67,6 +67,21 @@ document.addEventListener('turbolinks:load', () => {
             this.orderRows = [];
           }
 
+          if (user !== null) {
+            // Reload user to get latest credit balance
+            this.$http.get('/users/'+user.id).then((response) => {
+              const user = response.body;
+              this.$set(this.users, this.users.indexOf(user), user);
```

Comment:
Afaik indexOf works with objects that are already present in the list, right? Dont you need to pass an additional function to indexOf to define when a match is found?

Reply:
`indexOf` returns the index of the element (integer); `this.$set` then sets this index in the `this.users` array to `user`